testtweetme.py

```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import unittest
+import tweetme
+from birdy.twitter import UserClient
+
+
+class TestConfigFile(unittest.TestCase):
+    """
+    This test module will run only to test manage_token() method.
+    """
+
+    def test_1_nofile(self):
```

Comment:
No need to define tests having numbers. The name of the tests should be self-explanatory.

Reply:
The problem was I needed the tests to run in a specific order and unittest runs the test in the alphabetical order of naming. Hence, I had to include the numbers. I'll add a note about this too in order to self document about the importance of numbers.